packages/rs-drive-abci/src/execution/validation/state_transition/common/validate_state_transition_identity_signed/mod.rs

```diff
@@ -17,6 +17,9 @@ use crate::execution::validation::state_transition::common::validate_state_trans
 
 pub mod v0;
 
+#[allow(dead_code)]
+#[deprecated(note = "This function is marked as unused.")]
+#[allow(deprecated)]
```

Comment:
Why do we allow deprecated? Do we use deprecated code there?

Reply:
https://github.com/dashpay/platform/pull/1756#discussion_r1517134639

Reply:
Also; I do not believe there is any requirement to say `since`

Reply:
This one is not useful, deleting.